axon-explorer/start.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: start explorer
+  hosts: localhost
+  become: yes
+  become_method: sudo
+  vars_files:
+    - config.yml
+  tasks:
+  ###############################################################################################################
+    - name: star docker
+      shell: "docker run -p {{ localport }}:80 -e APP_NODE_URL={{ eth_rpc }} alethio/ethereum-lite-explorer"
+      become: yes
+      become_method: sudo
```

Comment:
ditto

Reply:
Use docker-compose to manage starting and stopping, using docker directly tends to forget to clean up containers。

Reply:
ok.